src/modules/esl-image/core/esl-image-iobserver.ts

```diff
@@ -21,6 +21,7 @@ function handleViewport(entry: IntersectionObserverEntry): void {
 
   // Check that entry is going to appear in the viewport area
   if (entry.isIntersecting || entry.intersectionRatio > 0) {
-    image.triggerLoad();
+    image.removeAttribute('lazy');
```

Comment:
Let's left the update visible for user
```suggestion
    image.$$attr('lazy', 'none');
```